shared/teams/team/subteams-tab/team-row/container.tsx

```diff
@@ -13,7 +13,7 @@ type OwnProps = {
 
 export default connect(
   (state, {teamname}: OwnProps) => ({
-    _newTeamRequests: state.teams.newTeamRequests || I.List(),
+    _newTeamRequests: state.teams.newTeamRequests || [],
```

Comment:
this will thrash, better to just allow undefined/null

Reply:
thanks, it turned out to be unneeded, `state.teams.newTeamRequests` is always set.